src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt

```diff
@@ -146,22 +198,21 @@ class ExternalDocRequestEndpointIntTest : IntegrationTestBase() {
             .andExpect(xpath("//ns3:Acknowledgement/Ack/TimeStamp", namespaces).exists())
             .andExpect(noFault())
 
-        checkMessagesOnQueue(0)
+        checkQueueIsEmpty()
         val date = LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE)
 
         checkS3Upload("fail-$date")
     }
 
-    private fun checkMessagesOnQueue(count: Int) {
-        val numberOfMessagesOnQueue = courtCaseEventsQueue?.sqsClient?.countMessagesOnQueue(courtCaseEventsQueue?.queueUrl!!)?.get()!!
-        assertThat(numberOfMessagesOnQueue).isEqualTo(count)
+    private fun checkQueueIsEmpty() {
+        val numberOfMessagesOnQueue = courtCasesQueue?.sqsClient?.countMessagesOnQueue(courtCasesQueue?.queueUrl!!)?.get()!!
+        assertThat(numberOfMessagesOnQueue).isEqualTo(0)
     }
 
-    private fun countMessagesOnQueue() = courtCaseEventsQueue?.sqsClient?.countMessagesOnQueue(courtCaseEventsQueue?.queueUrl!!)!!.get()
+    private fun countMessagesOnQueue() = courtCasesQueue?.sqsClient?.countMessagesOnQueue(courtCasesQueue?.queueUrl!!)!!.get()
 
     private fun checkS3Upload(fileNameStart: String) {
         val items = amazonS3.listObjects(ListObjectsRequest.builder().bucket(bucketName).build()).contents()
-        assertThat(items.size).isEqualTo(1)
```

Comment:
the uploads are named using a timestamp precide to seconds.
The FIFO test will therefore upload either 1 or 2 depending on whether it runs over the boundary of a second.
Removing this check seemed the simplest solution to this.